gnmi_cli_py/py_gnmicli.py

```diff
@@ -465,11 +468,25 @@ def main():
   user = args['username']
   password = args['password']
   form = args['format']
+  trigger_mem_spike = args['trigger_mem_spike']
   paths = _parse_path(_path_names(xpath))
   kwargs = {'root_cert': root_cert, 'cert_chain': cert_chain,
             'private_key': private_key}
   certs = _open_certs(**kwargs)
   creds = _build_creds(target, port, get_cert, certs, notls)
+
+  if trigger_mem_spike:
```

Comment:
>trigger_mem_spike [](http://example.com/codeflow?start=5&length=17)

I am thinking about the naming. For client point of view, this option is just a loop mode of subscribe.

How about the name `infinite_many_connections`, and it could be applied to any `mode`?

Reply:
Updated the option name. Do we want to implement and reproduce this memory spike issue for all modes such as `get`, `set` and `subscribe`?

Reply:
Yes

Reply:
Thanks for great suggestions!